fiat-github/src/main/java/com/netflix/spinnaker/fiat/roles/github/GithubTeamsUserRolesProvider.java

```diff
@@ -151,18 +162,38 @@ private boolean isMemberOfOrg(String username) {
   }
 
   private boolean isMemberOfTeam(Team t, String username) {
-    String ACTIVE = "active";
+    if (this.teamMembershipCache == null) {
+      initializeTeamMembershipCache();
```

Comment:
I missed in my initial review that this is an `InitializingBean`, which means you can add the `initializeTeamMembershipCache` call to the end of `afterPropertiesSet` method and remove this block. Sorry for not catching that the first time through.